src/main/java/org/opensearch/security/configuration/CompatConfig.java

```diff
@@ -69,7 +69,7 @@ public boolean restAuthEnabled() {
             } else {
                 final boolean restDynamicallyDisabled = dcm.isRestAuthDisabled();
                 if (isTraceEnabled) {
-                    log.trace("opendistro_security.dynamic.disable_rest_auth {}", restDynamicallyDisabled);
+                    log.trace("plugins.security.dynamic.disable_rest_auth {}", restDynamicallyDisabled);
```

Comment:
IMO, it refers to security config from the index, not settings

Reply:
Correct, this should be reverted.

Reply:
Done.